javascript/new-src/src/js/app.js

```diff
@@ -29,8 +29,6 @@ angular
   .factory('$ola', ['$http', '$window', 'OLA',
     function($http, $window, OLA) {
       'use strict';
-      // once olad supports json post data postEncode can go away
```

Comment:
We still need to do this don't we, so should leave the comment in, ideally with a TODO(Dave):